metrics_service/logger.py

```diff
@@ -0,0 +1,18 @@
+import logging
+import os
+import sys
+import warnings
+
+first = sys.argv[0]
+if first.endswith("manage.py"):
+    warnings.simplefilter(action="ignore", category=FutureWarning)
+
+
+def get_logger(name):
+    log_level = os.environ.get("METRICS_SERVICE_LOG_LEVEL", "INFO").upper()
```

Comment:
Maybe Im completely wrong here, but:

The log level env variable is read also in defaults.py, where it is written that:

These are default values that can be overridden by:
1. config/settings.yaml (Dynaconf)
2. Environment variables with METRICS_SERVICE_ prefix
3. /etc/ansible-automation-platform/settings.yaml (production)

So the get_logger should read value from dynaconf? (And I dont know how that works yet).

Because right now, the only source of truth for get_logger is env variable and dynaconf ment to have different sources with well defined priorites, right?